sdk/src/signer.rs

```diff
@@ -42,8 +50,10 @@ pub trait Signer {
     /// The default implementation will send the request to the URL
     /// provided by [`Self::time_authority_url()`], if any.
     fn send_timestamp_request(&self, message: &[u8]) -> Option<Result<Vec<u8>>> {
```

Comment:
Where is this called.   Does this function need to be in this trait or can this crate::time_stamp::default_rfc3161_request just be called where you call this function?

Reply:
`send_timestamp_request` was introduced in #311 and allows the implementor of `Signer` to completely replace the timestamp call. Unless we decide to remove that API, this needs to remain as-is.